main.cpp

```diff
@@ -1,162 +1,241 @@
-    return 0;    
-}
+#include <iostream>
+
+template<typename ...T>
+void ignoreUnused(T&&...) { }
+
+/*
+Project 2 - Part 1 / 1
+video: Chapter 2 - Part 3
+ Declarations Tasks
+
+ Create a branch named Part1
+
+ Purpose:  This project will teach you how to declare variables and free functions.  
+ This will be the first project where the code you write will be compiled and you will be responsible for making sure it compiles before submitting it for review.
+ 
+ 
+ 1) Write down the names of the 6 major primitive types available in C++  here:
+ 
+ int
+ float
+ char 
+ bool
+ double 
+ Unsigned int 
+ string //what about this one? 
+ 
+ 
+ 
+ 
+ 
+2) for each primitive type, write out 3 variable declarations inside the variableDeclaration() function on line 59.
+    a) give each variable declaration an initial value
+        - just ignore wchar_t. you do not need to declare 3 variables of type 'wchar_t'
+        - 'void' is a return type. you do not need to declare 3 variables of type 'void'.
+    b) at the end of the function, call ignoreUnused once and pass all of your variables to it. see line 71 for an example
+ 
+3) Declare 10 free functions
+    each declaration should have a random number of parameters in the function parameter list.
+    When naming your parameters, choose names that are relevant to the task implied by the function's name.
+
+4) add { ignoreUnused( ); } after each declaration in place of the closing semicolon
+5) pass each of your function parameters to the ignoreUnused function like you did in b)
+6) if your function returns something other than void, add 'return { };' at the end of it.
+7) provide default values for an arbitrary number of parameters in the function parameter list.
+
+8) consult the coding style guide found in the Readme.MD and adjust the formatting of your 10 functions.  At this point, you might have something that looks like this:
+    float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar ); return { }; }
+    This does not conform with the coding standard for this course (check the Readme.MD) and needs to be corrected
+    
+9) in the main function at the end: 
+    for each of those functions declared, 
+        a) write out how the function would look if called with correct arguments
+        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
+        c) pass the local variables to ignoreUnused() as you did in 2b)
+    see main() for an example of this.
+ 
+10) click the [run] button.  Clear up any errors or warnings as best you can.
+ */
+
+//2)
+void variableDeclarations()
+{
+    //example:
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int value = 1;
+    int param = 9;
+
+    float size = 0.3f;
+    float volume = 0.22222f;
+    float length = 0.27981f;
+
+    bool toggleSwitch = 0;
+    bool pressButton = 1;
+    bool touchSense = 1;
```

Comment:
Booleans need to be initialized with `true` or `false`. 
never initialize booleans with integers
- [ ] revise